<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutTopControls">
      <item>
       <widget class="QPushButton" name="processPushButton">
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/States/images/go.png</normaloff>:/States/images/go.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="processStopPushButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/States/images/media-playback-stop.png</normaloff>:/States/images/media-playback-stop.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="flagPushButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/States/images/flag-off.png</normaloff>
          <normalon>:/States/images/flag-on.png</normalon>:/States/images/flag-off.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelExperimentName">
        <property name="text">
         <string>Experiment name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="experimentNameLineEdit"/>
      </item>
      <item>
       <widget class="Line" name="lineTopSeparation">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelExperimentId">
        <property name="text">
         <string>Experiment id:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="experimentIdSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelArea">
        <property name="text">
         <string>Area:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="AreaLcdNumber">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <family>Courier</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="digitCount">
         <number>6</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelError">
        <property name="text">
         <string>Error:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="ErrorLcdNumber">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="digitCount">
         <number>6</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="backgroundBrush">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>24</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutBottomControls">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="maximum">
         <number>1</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="format">
         <string>%v/%m</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="lineBottomSeparation">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelRemoveNoiseIterations">
        <property name="text">
         <string>Remove Noise Iterations:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="RemoveNoiseIterationsSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxImageBinaryThreshold">
        <property name="text">
         <string>Image Binary Threshold</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="ImageBinaryThresholdHorizontalSlider">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>125</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1070</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Restart"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_Preferences"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="action_Prev"/>
   <addaction name="action_Play"/>
   <addaction name="action_Next"/>
   <addaction name="action_Plot"/>
  </widget>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/Action/images/document-open.png</normaloff>:/Action/images/document-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/Action/images/document-save.png</normaloff>:/Action/images/document-save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_Restart">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/States/images/edit-undo.png</normaloff>:/States/images/edit-undo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Restart</string>
   </property>
  </action>
  <action name="action_Next">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/Action/images/go-next.png</normaloff>:/Action/images/go-next.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Next</string>
   </property>
   <property name="toolTip">
    <string>Next Image</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="action_Prev">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/Action/images/go-previous.png</normaloff>:/Action/images/go-previous.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Prev</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="action_Play">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/States/images/media-playback-start.png</normaloff>:/States/images/media-playback-start.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="toolTip">
    <string>Automate</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_Plot">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/States/images/plot.png</normaloff>:/States/images/plot.png</iconset>
   </property>
   <property name="text">
    <string>Plot</string>
   </property>
   <property name="toolTip">
    <string>Show area plot</string>
   </property>
  </action>
  <action name="action_Preferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>534</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>processPushButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>RemoveNoiseIterationsSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>23</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>706</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>processPushButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>flagPushButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>23</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>processPushButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>ImageBinaryThresholdHorizontalSlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>23</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>771</x>
     <y>706</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>processPushButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkBoxImageBinaryThreshold</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>23</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>654</x>
     <y>706</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
